runtime/CSharp/src/Misc/Arrays.cs

```diff
@@ -2,46 +2,12 @@
  * Use of this file is governed by the BSD 3-clause license that
  * can be found in the LICENSE.txt file in the project root.
  */
-namespace Antlr4.Runtime.Sharpen
```

Comment:
oh i see. You are moving files around. I guess I'm just worried somebody might be using this but it's pretty deep inside so unlikely they are using right?

Reply:
Everything form `Sharpen` directory is utility classes and methods. Moreover, native API should be used instead of utility methods if possible, that's why most part of them can be removed. They should not be used out of ANTLR runtime.